www/js/banidb/realm-search.js

```diff
@@ -214,7 +251,7 @@ const getShabad = VerseID => (
  */
 const randomShabad = (SourceID = 'G') => (
   new Promise((resolve) => {
-    Realm.open(realmDB.realmVerseSchema)
+    Realm.open(realmSchemaDef)
```

Comment:
will opening the DB multiple times have any memory issues? doing some profiling looks like each time a search occurred the number of Realm objects increased (after a few searches by 1mb). Something to look out for incase we get feeback that the app gets progressively slower the more you use it.

Reply:
I'm glad you looked into this. Let's see what the best practice is.